### ISSUE TITLE
run amazon sample 0.5% run 3 5 epochs 

### ISSUE THREAD

**Issue by IsabelCanicoNeto** [jmribeiro/dsl-project#33]
amazon -sample -dataset_size 0.005 -epoch 5 -nrun 3 -tqdm
3000000lines [07:02, 7103.04lines/s]
3000000lines [11:29, 4350.91lines/s]
650000lines [02:32, 4268.28lines/s]
==== Split train.csv and test.csv files =====
.data/amazon_review_full_csv/train
Delete /train.csv
Rename /train_sample.csv to /train.csv
.data/amazon_review_full_csv/test
Delete /test.csv
Rename /test_sample.csv to /test.csv
 (Done) [13500 training samples]
*** run number  1 ****** Setting up han model on device cpu ***C:\ProgramData\Anaconda3\lib\site-packages\torch\nn\modules\rnn.py:51: UserWarning: dropout option adds dropout after all but last recurrent layer, so non-zero dropout expects num_layers greater than 1, but got dropout=0.1 and num_layers=1
  "num_layers={}".format(dropout, num_layers))
 (Done)
*** Setting up adam optimizer *** (Done)

# ##################### #
# Training han #
# ##################### #

Training epoch 1
100%|█████████▉| 210/211 [03:09<00:00,  1.30it/s]Training loss: 1.5328 
100%|██████████| 211/211 [03:10<00:00,  1.20it/s]

Valid acc: 0.3109 

Training epoch 2
100%|██████████| 211/211 [03:09<00:00,  1.32it/s]
Training loss: 1.4999 

Valid acc: 0.2962 
  0%|          | 0/211 [00:00<?, ?it/s]

Training epoch 3
100%|█████████▉| 210/211 [03:09<00:00,  1.20it/s]Training loss: 1.4807 

100%|██████████| 211/211 [03:09<00:00,  1.30it/s]
  0%|          | 0/211 [00:00<?, ?it/s]Valid acc: 0.3242 

Training epoch 4
100%|█████████▉| 210/211 [03:06<00:01,  1.17s/it]Training loss: 1.4652 

100%|██████████| 211/211 [03:07<00:00,  1.06s/it]
Valid acc: 0.3195 

  0%|          | 0/211 [00:00<?, ?it/s]Training epoch 5
100%|██████████| 211/211 [03:10<00:00,  1.05s/it]
Training loss: 1.4509 

Valid acc: 0.3189 

Final Test acc: 0.3223 

  0%|          | 0/211 [00:00<?, ?it/s] (Done)
*** Setting up adam optimizer *** (Done)

# ##################### #
# Training hsan #
# ##################### #

Training epoch 1
100%|██████████| 211/211 [03:16<00:00,  1.12it/s]
Training loss: 1.5706 

Valid acc: 0.3035 

  0%|          | 0/211 [00:00<?, ?it/s]Training epoch 2
100%|██████████| 211/211 [03:16<00:00,  1.19it/s]
Training loss: 1.5303 

Valid acc: 0.2882 
  0%|          | 0/211 [00:00<?, ?it/s]

Training epoch 3
100%|██████████| 211/211 [03:17<00:00,  1.15it/s]
Training loss: 1.5070 

  0%|          | 0/211 [00:00<?, ?it/s]Valid acc: 0.3202 

Training epoch 4
100%|██████████| 211/211 [03:18<00:00,  1.12s/it]
Training loss: 1.4894 

Valid acc: 0.3109 

Training epoch 5
100%|█████████▉| 210/211 [03:17<00:00,  1.04it/s]Training loss: 1.4748 

100%|██████████| 211/211 [03:19<00:00,  1.10s/it]
Valid acc: 0.3149 

Final Test acc: 0.3232 

*** Plotting validation accuracies and training losses *** (Done)

*** run number  2 ****** Setting up han model on device cpu ***C:\ProgramData\Anaconda3\lib\site-packages\torch\nn\modules\rnn.py:51: UserWarning: dropout option adds dropout after all but last recurrent layer, so non-zero dropout expects num_layers greater than 1, but got dropout=0.1 and num_layers=1
  "num_layers={}".format(dropout, num_layers))
  0%|          | 0/211 [00:00<?, ?it/s] (Done)
*** Setting up adam optimizer *** (Done)

# ##################### #
# Training han #
# ##################### #

Training epoch 1
100%|██████████| 211/211 [03:09<00:00,  1.26it/s]
Training loss: 1.5325 

Valid acc: 0.3029 

  0%|          | 0/211 [00:00<?, ?it/s]Training epoch 2
100%|██████████| 211/211 [03:08<00:00,  1.24it/s]
Training loss: 1.5008 

Valid acc: 0.3022 
  0%|          | 0/211 [00:00<?, ?it/s]

Training epoch 3
100%|█████████▉| 210/211 [03:09<00:00,  1.10it/s]Training loss: 1.4810 

100%|██████████| 211/211 [03:09<00:00,  1.22it/s]
  0%|          | 0/211 [00:00<?, ?it/s]Valid acc: 0.3162 

Training epoch 4
100%|█████████▉| 210/211 [03:08<00:01,  1.19s/it]Training loss: 1.4653 

100%|██████████| 211/211 [03:09<00:00,  1.07s/it]
  0%|          | 0/211 [00:00<?, ?it/s]Valid acc: 0.3269 

Training epoch 5
100%|██████████| 211/211 [03:10<00:00,  1.06s/it]
Training loss: 1.4508 

Valid acc: 0.3229 

Final Test acc: 0.3318 

*** Setting up hsan model on device cpu *** (Done)
*** Setting up adam optimizer *** (Done)

# ##################### #
# Training hsan #
# ##################### #
  0%|          | 0/211 [00:00<?, ?it/s]
Training epoch 1
100%|█████████▉| 210/211 [03:16<00:00,  1.29it/s]Training loss: 1.5455 

100%|██████████| 211/211 [03:17<00:00,  1.22it/s]
Valid acc: 0.2955 
  0%|          | 0/211 [00:00<?, ?it/s]

Training epoch 2
100%|█████████▉| 210/211 [03:17<00:00,  1.15it/s]Training loss: 1.5132 

100%|██████████| 211/211 [03:18<00:00,  1.17it/s]
  0%|          | 0/211 [00:00<?, ?it/s]Valid acc: 0.2849 

Training epoch 3
100%|██████████| 211/211 [03:31<00:00,  1.13it/s]
Training loss: 1.4951 

  0%|          | 0/211 [00:00<?, ?it/s]Valid acc: 0.3189 

Training epoch 4
100%|██████████| 211/211 [03:28<00:00,  1.18s/it]
Training loss: 1.4798 

Valid acc: 0.3115 

  0%|          | 0/211 [00:00<?, ?it/s]
Training epoch 5
100%|██████████| 211/211 [03:34<00:00,  1.10s/it]
Training loss: 1.4666 

Valid acc: 0.3289 

Final Test acc: 0.3204 

*** Plotting validation accuracies and training losses *** (Done)

*** run number  3 ****** Setting up han model on device cpu ***C:\ProgramData\Anaconda3\lib\site-packages\torch\nn\modules\rnn.py:51: UserWarning: dropout option adds dropout after all but last recurrent layer, so non-zero dropout expects num_layers greater than 1, but got dropout=0.1 and num_layers=1
  "num_layers={}".format(dropout, num_layers))
 (Done)
*** Setting up adam optimizer *** (Done)

# ##################### #
# Training han #
# ##################### #

Training epoch 1
100%|█████████▉| 210/211 [03:12<00:00,  1.20it/s]Training loss: 1.5534 

100%|██████████| 211/211 [03:13<00:00,  1.29it/s]
Valid acc: 0.3042 

Training epoch 2
100%|██████████| 211/211 [03:11<00:00,  1.25it/s]Training loss: 1.5144 

Valid acc: 0.2915 

Training epoch 3
100%|██████████| 211/211 [03:16<00:00,  1.17it/s]
Training loss: 1.4915 

Valid acc: 0.3289 

Training epoch 4
100%|██████████| 211/211 [03:11<00:00,  1.06s/it]
Training loss: 1.4739 

Valid acc: 0.3135 
  0%|          | 0/211 [00:00<?, ?it/s]

Training epoch 5
100%|██████████| 211/211 [03:13<00:00,  1.01it/s]
Training loss: 1.4577 

Valid acc: 0.3336 

Final Test acc: 0.3278 

  0%|          | 0/211 [00:00<?, ?it/s] (Done)
*** Setting up adam optimizer *** (Done)

# ##################### #
# Training hsan #
# ##################### #

Training epoch 1
100%|██████████| 211/211 [03:19<00:00,  1.19it/s]
Training loss: 1.5461 

  0%|          | 0/211 [00:00<?, ?it/s]Valid acc: 0.2929 

Training epoch 2
100%|█████████▉| 210/211 [03:20<00:00,  1.14it/s]Training loss: 1.5126 

100%|██████████| 211/211 [03:21<00:00,  1.16it/s]
  0%|          | 0/211 [00:00<?, ?it/s]Valid acc: 0.2995 

Training epoch 3
100%|██████████| 211/211 [03:27<00:00,  1.00it/s]
Training loss: 1.4927 

Valid acc: 0.3235 

Training epoch 4
100%|█████████▉| 210/211 [03:33<00:01,  1.27s/it]Training loss: 1.4778 

100%|██████████| 211/211 [03:34<00:00,  1.15s/it]
Valid acc: 0.3075 

Training epoch 5
100%|██████████| 211/211 [03:50<00:00,  1.54s/it]
Training loss: 1.4641 

Valid acc: 0.3202 

Final Test acc: 0.3284 

*** Plotting validation accuracies and training losses *** (Done)

Process finished with exit code 0

ficheiros 3/1 15:00
